osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -34,7 +34,7 @@ public partial class ScoresContainer : BeatmapSetLayoutSection
         private readonly IBindable<APIUser> user = new Bindable<APIUser>();
 
         private readonly Box background;
-        private readonly ScoreTable scoreTable;
+        public readonly ScoreTable ScoreTable;
```

Comment:
Why is this public? For the test?

If it's for the test, then please make private again and use `.ChildrenOfType<>()` in the test instead.

Reply:
It was only for the test